pydash/pydash_web/controller/dashboards.py

```diff
@@ -74,12 +75,18 @@ def endpoint_dict(endpoint):
         }
     endpoints = [endpoint_dict(endpoint) for endpoint in dashboard.endpoints.values()]
 
-    return {
+    dash_dict = {
         'id': dashboard.id,
         'url': dashboard.url,
         'endpoints': endpoints
     }
 
+    for state in DashboardState:
+        if dashboard.state == state and str(state.name).split("_")[-1] == "failure":
```

Comment:
Can't you just check `dashboard.state.name.split('_')[-1] == 'failure'`? :stuck_out_tongue: 

In the future, we should probably add a method to check if the state is an error state (because checking the name of the state is hacky)